rafiki/client/client.py

```diff
@@ -230,20 +235,20 @@ def download_model_file(self, name, model_file_path):
     ####################################
     
     def create_train_job(self, app, task, train_dataset_uri,
-                        test_dataset_uri, budget=None):
+                        test_dataset_uri, models):
         '''
         Creates and starts a train job on Rafiki. 
         A train job is uniquely identified by its associated app and the app version (returned in output).
         
-        Only admins & app developers can manage train jobs.
+        Only admins, model developers and app developers can manage train jobs.
 
         :param str app: Name of the app associated with the train job
         :param str task: Task associated with the train job, 
             the train job will train models associated with the task
         :param str train_dataset_uri: URI of the train dataset in a format specified by the task
         :param str test_dataset_uri: URI of the test (development) dataset in a format specified by the task
-        :param budget: budget for the train job
-        :type budget: dict[:class:`rafiki.constants.BudgetType`, int]
+        :param models: models to use for the train job. List of dictionaries containing model name and budget.
```

Comment:
Should specify the format of params `models` a bit more clearly (e.g. give example in docs?)